### PR TITLE
add metrics about rejected logappend msg count

### PR DESCRIPTION
**Pull request by tier-cap**
Signed-off-by: tier-cap <zhengxiaojin@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 5kbpers — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 - gRPC message error：每个 TiKV 实例上 gRPC 消息发生错误的个数
 - Leader drop：每个 TiKV 实例上 drop leader 的个数
 - Leader missing：每个 TiKV 实例上 missing leader 的个数
+- Log Replication Rejected: 每个 TiKV 实例上由于内存不足而拒绝 log append message 的个数
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:44]:
```suggestion
- Log Replication Rejected：每个 TiKV 实例上由于内存不足而拒绝追加 log 的消息个数
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: grafana-tikv-dashboard.md (hunk 1 above)
Comment (tier-cap) [reply grafana-tikv-dashboard.md:44]: Log Replication Rejected：每个 TiKV 实例上由于内存不足而拒绝 logappend 消息的个数

**Review by Oreoxmt — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 - gRPC message error：每个 TiKV 实例上 gRPC 消息发生错误的个数
 - Leader drop：每个 TiKV 实例上 drop leader 的个数
 - Leader missing：每个 TiKV 实例上 missing leader 的个数
+- Log Replication Rejected：每个 TiKV 实例上由于内存不足而拒绝 logappend 消息的个数
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:44]: 6.1 以前的版本中，Grafana 面板上是没有这个指标的吗？

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -41,6 +41,7 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
 
 ![TiKV Dashboard - Errors metrics](/media/tikv-dashboard-errors.png)
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:46]:
```suggestion
![TiKV Dashboard - Errors metrics](/media/tikv-dashboard-errors-v610.png)
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**